### PR TITLE
Cleanup.

### PR DESCRIPTION
**Pull request by dkorolev**

### CODE REVIEW

File: Karl/karl.h [hunk 1]
```diff
@@ -39,13 +39,18 @@ SOFTWARE.
 // The conventional wisdom is that Karl can start with both 1) and 2) missing. After one keepalive cycle,
 // which is under half a minute, it would regain the state of the fleet, as long as all keepalives go to it.
 
-// NOTE: Local `current_build.h` must be included before Karl/Claire headers.
+// NOTE(mzhurovich): Local `current_build.h` must be included before Karl/Claire headers.
+// NOTE(dkorolev): Why the hell? :-)
```
Comment (mzhurovich) [inline Karl/karl.h]: What did you mean by `Why the hell?`?

Comment (dkorolev) [inline Karl/karl.h]: What exactly is wrong with `current_build.h` being `#include`-d directly and unconditionally from both `karl.h` and `claire.h`?

Comment (mzhurovich) [inline Karl/karl.h]: I'm not sure I understand you. Which `current_build.h` do you mean? The one from from the `Karl` directory is irrelevant to the project using Karl.

**Comment by dkorolev** [comment]
Up!

@mzhurovich, bash quotes have been defeated, and both top-level `make test` and `make check` pass on a freshly cloned repositories cloned into `~/make test/cur rent` and `~/make check/cur rent` respectively.

PTAL.

**Pull request merged**